daemon/daemon.go

```diff
@@ -575,6 +576,12 @@ func (d *Daemon) Stop() error {
 		}
 		// wait for reboot to happen
 		logger.Noticef("Waiting for system reboot")
+		signal.Stop(sigCh)
```

Comment:
before adding the close probably it would have worked anyway, but maybe I would add a if sigCh != nil around all this, so the tests that don't care can do Stop(nil)  and also it seems a nicer interface in itself making sigCh optional